src/Microsoft.Health.Dicom.Api/Features/Filters/DicomRequestContextRouteDataPopulatingFilterAttribute.cs

```diff
@@ -39,7 +39,15 @@ public override void OnActionExecuting(ActionExecutingContext context)
         IDicomRequestContext dicomRequestContext = _dicomRequestContextAccessor.RequestContext;
         dicomRequestContext.RouteName = context.ActionDescriptor?.AttributeRouteInfo?.Name;
 
-        dicomRequestContext.Version = context.HttpContext.GetRequestedApiVersion()?.MajorVersion;
+        int? version = null;
```

Comment:
Can we use `GetMajorRequestedApiVersion`?

Reply:
The behaviour is slightly different - not sure how much it matters. Here we will have a `null` version if none can be found, and `GetMajorRequestedApiVersion` will return 1